src/test/infrastructure/elasticsearch/elasticsearch-values.yaml

```diff
@@ -1,27 +1,15 @@
----
+persistence:
+  enabled: false
 
-global:
-  coordinating:
-    name: coo
+secret:
```

Comment:
Auth is enabled by default. If we want to we may keep it enabled and pass secret name and keys to bitbucket.